src/js/menu/operations/TurfOperation.js

```diff
@@ -10,7 +11,16 @@ L.DNC.TurfOperation = L.DNC.Operation.extend({
     },
 
     execute: function () {
-        var layers = L.DNC.layerlist.selection.list;
+        /*
+        **
+        **  TODO: this is the type of referencing
+        **  that feels like it should be avoided.
+        **  it is the only reference left and it cannot
+        **  be factored out until we potentially
+        **  revisit how MenBar, Menu and Operation work together
```

Comment:
I haven't looked at the relationship in depth enough to guess what needs to change now. But in particular it feels like the `L.DNC.MenuBar` and it's inlaws own the selection relationship more than `L.DNC.LayerList` does.

So a quick win in terms of this reference might be to have `L.DNC.LayerList` fire events for add and remove [here](https://github.com/cugos/drop-n-chop/blob/remove-reflexive-refs/src/js/layerlist/LayerList.js#L185-L192) and `L.DNC.MenuBar` and inlaws listen for those and hold the selection internally. 

That's a quick guess shooting from the hip, so not sure if that would work.